yt_dlp/extractor/tempo.py

```diff
@@ -1,20 +1,20 @@
 from .common import InfoExtractor
-from ..utils import int_or_none, parse_iso8601, str_or_none, traverse_obj
+from ..utils import try_call
 
 
 class TempoIE(InfoExtractor):
     _VALID_URL = r'https?://video\.tempo\.co/\w+/\d+/(?P<id>[\w-]+)'
     _TESTS = [{
         'url': 'https://video.tempo.co/read/30058/anies-baswedan-ajukan-banding-putusan-ptun-batalkan-ump-dki',
         'info_dict': {
-            'id': '2144438',
```

Comment:
The code for extracting id looks same. Why does this change?

Reply:
I had no idea why the `id` change. I'm quite sure it was correct `id` and somehow they change the id

Reply:
Resolved this, as this seems change in tempo/ivx backend